doc/newsfragments/custom_style_of_assertion_header.feature.rst

```diff
@@ -0,0 +1 @@
+* Support custom style (e.g. background color) of assertion header on web UI.
```

Comment:
Announcement need to link to example / doc, so that user know how to use this feature.

Reply:
added some new marks.

* Support custom style (e.g. background color) of assertion header on web UI, refer to :ref:`document <Styling_Assertions>`